### ISSUE TITLE
Connection reset by peer ?

### ISSUE THREAD

**Issue by tienhm0202** [fschr/simpletcp#10]
Sometimes I got my server stop by error `Connection reset by peer` at `serversocket.py`, Line 76
```python
data = sock.recv(self.recv_bytes)
```
How can I make server run forever and ignore this error? I'm trying to use your lib on production. Please give me an advice 😆

**Comment by fschr** [comment]
@tienhm0202 You could surround that part of the code with a try/catch and handle it gracefully (i.e., continue running). If you find a solution that you think generalizes, you could submit a pull request.

**Comment by fschr** [comment]
@tienhm0202 I'm working on a fix for this right now. Could you post your entire error message?

**Comment by tienhm0202** [comment]
This is Trace Back:
```
Traceback (most recent call last):
  File "tcp/tcp_main.py", line 38, in <module>
    server.run()
  File "/home/ubuntu/giamsat/gs/lib/python3.5/site-packages/simpletcp/serversocket.py", line 76, in run
    data = sock.recv(self.recv_bytes)
ConnectionResetError: [Errno 104] Connection reset by peer
```

And this is my server:
```python
```

**Comment by tienhm0202** [comment]
@fschr I tried code like this
```python
while True:
try:
   server = TCPServer(HOST, PORT, handle_tcp_non_blocking, recv_bytes=1024)
   server.run()
except KeyboardInterrupt:
    print("Stopped by admin")
except Exception as e:
    continue
```
But it's not work. Look like this socket is not clear and I got `Address already used` error.

**Comment by fschr** [comment]
@tienhm0202 If the socket is already in use, try a different port. You can see which processes are already using the port that you're trying to bind to with `netstat` or `lsof` or the /proc/ filesystem. Also, can you look at my PR? https://github.com/fschr/simpletcp/pull/11

**Comment by tienhm0202** [comment]
Fixed at PR #11
